ledger/ledger-api-bench-tool/src/test/suite/scala/com/daml/ledger/api/benchtool/config/WorkflowConfigParserSpec.scala

```diff
@@ -56,6 +56,8 @@ class WorkflowConfigParserSpec extends AnyWordSpec with Matchers {
           |        templates:
           |         - Foo1
           |         - Foo3
+          |        interfaces:
```

Comment:
Is the new 'interfaces:' config property required to be present even if the don't want to specify any interfaces in the request?
Consider making it optional, as it would save us the burden of updating all the existing benchmark yml definitions in on oem kit repo

Reply:
Made them optional